pkg/adapters/rules/kubernetes/rule.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2016 IBM Corporation
```

Comment:
Can we call this routingrules or config? rules is very ambiguous and it does not give any idea as to what is under this package tree.